scripts/bootstrap.js

```diff
@@ -83,7 +83,7 @@ async.parallelLimit(packages.map(function (root) {
             }, null, "  "), function (err) {
               if (err) return done(err);
 
-              fs.writeFile(linkDest + "/index.js", 'module.exports = require("' + linkSrc + '");', done);
+              fs.writeFile(linkDest + "/index.js", 'module.exports = require(' + JSON.stringify(linkSrc) + ');', done);
```

Comment:
On Windows, path names contain backslashes and those need to be escaped. `JSON.stringify` is just an easy way to obtain a valid quoted and escaped JSON string, which also happens to be a valid JS string.